plugins/com.google.cloud.tools.eclipse.dataflow.core/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,43 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Plugin
```

Comment:
Done. Moved to plugin.properties, and as in at least one of the appengine plugins.